packages/utils/test/is.test.ts

```diff
@@ -134,3 +135,19 @@ describe('isNaN()', () => {
     expect(isNaN(new Date())).toEqual(false);
   });
 });
+
+describe('isVueViewModel()', () => {
+  test('should work as advertised', () => {
+    expect(isVueViewModel({ _isVue: true })).toEqual(true);
+    expect(isVueViewModel({ __isVue: true })).toEqual(true);
```

Comment:
super basic request, just to be safe - can we add a test for the inverse, i.e. show that an object without the `_isVue` makes the function return `false`?